regression/cbmc-library/alloca-03/main.c

```diff
@@ -0,0 +1,18 @@
+#include <stdlib.h>
+
+#ifdef _WIN32
+void *alloca(size_t alloca_size);
+#endif
+
+typedef int *int_ptr;
+
+int main()
```

Comment:
Do we need a second test that checks deallocation when returning from the function?

Reply:
Good point, I'll expand the test to that effect (even though I do think we cover this case in the test `alloca-02`).